CMakeLists.txt

```diff
@@ -127,11 +144,19 @@ if (CONDA_BUILD)
         set (MAP_IMAGES_PATH $ENV{MAP_IMAGES_PATH})
     endif()
 else()
+    message("THIRD_PARTY_DIR = ${THIRD_PARTY_DIR}")
     include_directories (SYSTEM ${THIRD_PARTY_INC_DIR})
     include_directories (SYSTEM ${THIRD_PARTY_INC_DIR}/freetype2)
     link_directories (${THIRD_PARTY_LIB_DIR})
     link_directories (${PYTHONPATH})
+
     list (APPEND CMAKE_PREFIX_PATH ${THIRD_PARTY_LIB_DIR})
+    list (APPEND CMAKE_PREFIX_PATH ${THIRD_PARTY_DIR})
+    list (APPEND CMAKE_PREFIX_PATH ${THIRD_PARTY_DIR}/HDF_Group/HDF5/1.12.2/lib)
```

Comment:
I don't believe we are linking against these?

Reply:
Hmm I'm not sure what you mean.  We are linking against HDF5, but we are currently  using their binary installer which doesn't install into ${THIRD_PARTY_LIB_DIR}, rather in ${THIRD_PARTY_DIR}/HDF_Group/HDF5/1.12.2/lib.

Reply:
I thought this was for the plugins. If HDF5 can be installed in an arbitrary location why not just install it with the rest of the libraries?

Reply:
> I thought this was for the plugins.

No, this is for the entirety of HDF5.

I'm currently using their binary installer which has a weird mangled path.  Ideally we would be compiling it a more normal path but HDF5 is one of the harder libraries to deal with, alongside cpython and ospray.

I agree that compiling it would be ideal, but it comes with a rat's nest of problems that I haven't had have time to suss out.